docs/en/observability/logs-ecs-application.asciidoc

```diff
@@ -0,0 +1,206 @@
+
+include::tab-widgets/logs/filebeat-setup-widget.asciidoc[]
+
+From the {filebeat} installation directory, start filebeat by running the command that aligns with your system:
+
+include::{beats-root}/libbeat/docs/tab-widgets/start-widget-filebeat.asciidoc[]
+
+[discrete]
+[[ingest-ecs-logs-with-agent]]
+=== Ingest logs with {agent}
+
+Add the custom logs integration to ingest and centrally manage your logs using {agent} and {fleet}:
+
+[discrete]
+[[step-1-add-the-custom-logs-integration-to-your-project-ecs]]
+=== Add the custom logs integration to your project
+
+To add the custom logs integration to your project:
+
+. From your project, navigate to **Project Settings →   Integrations**.
+. Type `custom` in the search bar and select **Custom Logs**.
+. Click **Install {agent}** at the bottom of the page, and follow the instructions for your system to install the {agent}.
+. After installing the {agent}, click **Save and continue** to configure the integration from the **Add Custom Logs integration** page.
+. Give your integration a meaningful name and description.
+. Add the **Log file path**. For example, `/var/log/your-logs.log`.
+. Click **Advanced options**.
+. In the **Processors** text box, add the following YAML configuration to add processors that enhance your data. See {filebeat-ref}/filtering-and-enhancing-data.html[processors] to learn more.
++
+[source,yaml]
+----
+processors:
+  - add_host_metadata: \~
+  - add_cloud_metadata: \~
+  - add_docker_metadata: \~
+  - add_kubernetes_metadata: \~
+----
+. Under **Custom configurations**, add the following YAML configuration to collect data.
++
+[source,yaml]
+----
+  json:
+    overwrite_keys: true <1>
+    add_error_key: true <2>
+    expand_keys: true <3>
+    keys_under_root: true <4>
+  fields_under_root: true <5>
+  fields:
+    service.name: your_service_name [^6]
+    service.version: your_service_version [^6]
+    service.environment: your_service_environment [^6]
+----
++
+<1>: Values from the decoded JSON object overwrite the fields that {agent} normally adds (type, source, offset, etc.) in case of conflicts.
+<2>: {agent} adds an "error.message" and "error.type: json" key in case of JSON unmarshalling errors.
+<3>: {agent} will recursively de-dot keys in the decoded JSON, and expand them into a hierarchical object structure.
+<4>: By default, the decoded JSON is placed under a "json" key in the output document. When set to `true`, the keys are copied top level in the output document.
+<5>: When set to `true`, custom fields are stored as top-level fields in the output document instead of being grouped under a fields sub-dictionary.
+<6>: The `service.name` (required), `service.version` (optional), and `service.environment` (optional) of the service you're collecting logs from, used for log correlation.
+. An agent policy is created that defines the data your {agent} collects.
```

Comment:
This seems like it's in the wrong order maybe? Also I think a step should tell someone to do something, not explain what happens after someone has done something.